internal/integration_test/fuzz/fuzz/fuzz_targets/validation.rs

```diff
@@ -0,0 +1,29 @@
+#![no_main]
+
+use libfuzzer_sys::arbitrary::{Result, Unstructured};
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &[u8]| {
+    drop(run(data));
+});
+
+fn run(data: &[u8]) -> Result<()> {
+    // Create the random source.
+    let mut u = Unstructured::new(data);
+
+    // Generate the random module via wasm-smith, but MaybeInvalidModule.
+    let module: wasm_smith::MaybeInvalidModule = u.arbitrary()?;
```

Comment:
so this isn't sure if it will make an invalid one or not.. interesting :D

Reply:
yeah not sure until we pass it to wazero :D